docs/mmctl_command_archive.md

```diff
@@ -1,25 +1,25 @@
-## mmctl command delete
+## mmctl command archive
 
-Delete a slash command
+Archive a slash command
 
 ### Synopsis
 
-Delete a slash command. Commands can be specified by command ID.
+Archive a slash command. Commands can be specified by command ID.
 
 ```
-mmctl command delete [flags]
+mmctl command archive [flags]
```

Comment:
I know this is already merged, but for consistency with the other commands shouldn't this be 
```
mmctl command archive commandID [flags]
```
?  @ethervoid   @mgdelacroix